installer/install.iss

```diff
@@ -740,8 +740,13 @@ begin
 
     TmpFile:=ExpandConstant('{tmp}\git-config-get.txt');
     if not Exec(ExpandConstant('{cmd}'),'/D /C .\{#MINGW_BITNESS}\bin\git.exe config -l -z '+ExtraOptions+' >'+#34+TmpFile+#34,
-                AppDir,SW_HIDE,ewWaitUntilTerminated,i) then
-        LogError('Unable to get system config');
+                AppDir,SW_HIDE,ewWaitUntilTerminated,i) then begin
+        if FileExists(AppDir+ExpandConstant('\{#MINGW_BITNESS}\bin\git.exe')) then
```

Comment:
It feels a little odd, that we try to execute `{#MINGW_BITNESS}\bin\git.exe` and check wether it exists afterwards, but it should work fine.

Reply:
Right. I did want to favor the common code path, though, and that would be: the call succeeds.